README.rst

```diff
@@ -93,6 +93,20 @@ Alternately, one can connect with NetCat: ``nc -C 127.0.0.1 4444`` or Socat: ``s
 tcp:127.0.0.1:4444`` (for line editing and history support).  When finished debugging, either exit 
 the debugger, or press Control-c.
 
+Integration with breakpoint() in Python 3.8+
+============================================
+
+If you are using Python 3.8 you can specify that ``breakpoint()`` should invoke
+remote_pdb. In this case, you may use environment variables to specify the
+host and port that should be used:
+
+.. code:: bash
+
+    PYTHONBREAKPOINT=remote_pdb.set_trace REMOTE_PDB_HOST=0.0.0.0 REMOTE_PDB_PORT=10000 python script.py
+
+Omitting either of these environment variables will use the default logic from above.
```

Comment:
Hmmm. I'd reword this somehow, it doesn't sound right. Perhaps just state what the defaults are?

Reply:
I've reworked the whole section to try and be more explicit and be closer to the voice you used in the other parts of the readme. I'd originally tried to make this a light touch section to avoid restating things, but your idea of being explicit works for me.